migration/migration.go

```diff
@@ -583,7 +583,7 @@ func BootstrapWorkItemLinking(ctx context.Context, linkCatRepo *link.GormWorkIte
 		ID:             link.SystemWorkItemLinkTypeParentChildID,
 		Name:           "Parent child item",
 		Description:    &parentingDesc,
-		Topology:       link.TopologyNetwork,
+		Topology:       link.TopologyTree,
```

Comment:
Can we test these migration?

Reply:
Well, we have checks for the link type repository's `Save` and `Create` methods in place. And `createOrUpdateWorkItemLinkType` does call those methods. I think this is safe enough.

Reply:
Ok.